api/Makefile

```diff
@@ -1,3 +1,12 @@
+AIR_BIN=bin/air
+
+run:
+	if [ ! -f $(AIR_BIN) ]; then \
+		echo "Installing Air locally..."; \
+		go build -o $(AIR_BIN) github.com/air-verse/air; \
```

Comment:
I think air should be globally installed not inside the repo.

Reply:
Yes @plutov, I agree, but in case if it's not installed globally, formulosity can still be ran in dev mode if we install it locally in this project itself.

But, I'd prefer for global installation, what do you suggest? Should we make it compulsory for AIR to be installed gloabally?